kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -219,6 +222,18 @@ public static ConfigDef newConfigDef() {
           "S3 Bucket"
       );
 
+      configDef.define(
+          S3_OBJECT_TAGGING_CONFIG,
+          Type.BOOLEAN,
+          S3_OBJECT_TAGGING_DEFAULT,
+          Importance.LOW,
+          "Tag S3 objects with offsets & record count.",
```

Comment:
How about: 
"Tag S3 objects with start and end offsets, as well as record count"
cc @tobydrake7

Reply:
@kkonstantine looks good 👍